javascript/view.js

```diff
@@ -0,0 +1,145 @@
+const displayProductsView = {
+    init(DISPLAY_PRODUCTS_CONTENT){
+        this.productsToBeSold = document.getElementById('productsToBeSold');
+        this.DISPLAY_PRODUCTS_CONTENT = DISPLAY_PRODUCTS_CONTENT;
```

Comment:
This way you are creating a copy of the model. This way there are multiple source of truth being created. Rather than doing this pull this data by the help of controller